src/shogun/statistics/QuadraticTimeMMD.cpp

```diff
@@ -1023,7 +1024,7 @@ SGVector<float64_t> CQuadraticTimeMMD::sample_null_spectrum_DEPRECATED(
 
 	/* when m=n, return m*MMD^2 instead */
 	if (m==n)
-		null_samples.scale(0.5);
+		linalg::scale(null_samples, null_samples, 0.5);
```

Comment:
hey

just curious, can't there be a convenience method for the case where both arugments are the same?